### ISSUE TITLE
[MAS1.4.3] [Visual Requirements-Help->Get started with channels] Luminosity Ratio is less than 4.5:1 “Jump to bot state inspection, The latest emulator,ngrok,update your bots code, Azure portal…etc”

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1807]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 “Jump to bot state inspection, The latest emulator, ngrok, update your bots code, Azure portal…etc”

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Jump to bot state inspection, The latest emulator,ngrok,update your bots code, Azure portal…etc”

**Note:** Same issue exits for all links present in the Get started with the bot inspector screen 
Same issue repro for below path 
Bot Explorer->Services->+symbol->Add Dispatch
Please Refer the attachment:MAS1 4 3_Add a Dispatch.PNG
Bot Explorer->Services->+symbol->Add Dispatch->Connect to a dispatch model manually
Please refer the attachment:MAS1 4 3_Dispatch.png
Bot Explorer->Services->+symbol->Add language understanding-> add a LUIS app manually
Please refer the attachment:MAS1 4 3_Luis.png

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Help menu and select it 
3.Navigate to Get started with channels(bot inspector) and select it 
4.Dialog gets opened
6.Open 'Colour Contrast Analyser' application.
7.Verify whether Luminosity Ratio is less than 4.5:1 for “Jump to bot state inspection, The latest emulator,ngrok,update your bots code, Azure portal…etc” “links or not

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![MAS1 4 3_File](https://user-images.githubusercontent.com/53518994/64061386-a6f5d480-cbf7-11e9-8103-d073d0001933.png)
![MAS1 4 3_Add a Dispatch](https://user-images.githubusercontent.com/53518994/64062511-c8a98880-cc04-11e9-8289-ad908889c9d9.png)
![MAS1 4 3_Dispatch](https://user-images.githubusercontent.com/53518994/64062525-16be8c00-cc05-11e9-9cb1-a69f57cdc7cb.png)
![MAS1 4 3_Luis](https://user-images.githubusercontent.com/53518994/64062554-73ba4200-cc05-11e9-9841-689750db373b.png)

**Comment by tonyanziano** [comment]
This is confirmed as fixed in master:

![image](https://user-images.githubusercontent.com/3452012/64458805-2c233280-d0ab-11e9-92ef-7c511f4b8209.png)

![image](https://user-images.githubusercontent.com/3452012/64458887-6987c000-d0ab-11e9-9beb-7ea31ed8f4c6.png)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator Version 4.5.2-78154 and issue got  fixed now Luminosity ratio is 4.5:1 for controls Hence we are Closing  this bug.
